### PR TITLE
VAR-164 | Staff cannot see normal reservations

### PR DESCRIPTION
**Pull request by midori0507**
- Staff now able to see all reservations
- Staff which have permissions can perform reservation edit, for example changing reservation state, comments..
- Staff without permission only can open modal, see informations. Cant comment, cant edit

### CODE REVIEW

File: src/domain/reservation/manage/list/ManageReservationsList.js [hunk 1]
```diff
@@ -68,6 +69,8 @@ const ManageReservationsList = ({
                     onEditReservation={onEditReservation}
                     onInfoClick={() => onInfoClick(reservation)}
                     reservation={normalizedReservation}
+                    userCanCancel={reservationUtils.canUserCancelReservation(reservation)}
```
Comment (Nizar-Rahme) [inline src/domain/reservation/manage/list/ManageReservationsList.js]:
```suggestion
                    userCanDelete={reservationUtils.canUserCancelReservation(reservation)}
```

Comment (midori0507) [reply src/domain/reservation/manage/list/ManageReservationsList.js]: i know that's dumb but `Cancel` should go along with `Cancel`..
Delete is not one of reservation state..

Comment (Nizar-Rahme) [reply src/domain/reservation/manage/list/ManageReservationsList.js]: I saw the `propTypes` in [ManageReservationsDropdown.js](https://github.com/City-of-Helsinki/varaamo/pull/1004/files#diff-dbe147be8b0229268f557adf4ec40370), they don't have `userCanCancel`, instead, they have `userCanDelete`.

Comment (midori0507) [reply src/domain/reservation/manage/list/ManageReservationsList.js]: Hah, will fix 👍

**Pull request merged**